test/openAPI/features/support/helpers/helpers.js

```diff
@@ -194,6 +194,23 @@ module.exports = {
     },
     required: ['query', 'write'],
   },
+  // update-or-create
+  dataUpdateOrCreateEndpoint:
+    'data/{registryname}/{versionnumber}/update-or-create',
+  dataUpdateOrCreateResponseSchema: {
+    type: 'object',
+    properties: {
+      content: {
+        type: 'object',
+        properties: {
+          ID: { type: 'string' },
+          FirstName: { type: 'string' },
+          LastName: { type: 'string' },
+          BirthCertificateID: { type: 'string' },
+        },
+      },
```

Comment:
This part is already written in `dataExampleSchema` in the same file ;)
Also, should those properties be required? In another endpoints there are